### ISSUE TITLE
[Feature Request] Clocks Applet

### ISSUE THREAD

**Issue by Justinzobel** [ikeydoherty/budgie-next#11]
To view multiple timezones in the ncenter

**Comment by ikeydoherty** [comment]
So an enhanced clock applet? On account of we already have a clock applet :]

**Comment by Justinzobel** [comment]
One for ncenter that I can use to display multiple clocks

e.g:
Adelaide 9:33PM
London 1:54am
San Francisco 5:15Pm

**Comment by JoshStrobl** [comment]
@ikeydoherty Maybe this could tie into `gnome-clocks`?

**Comment by Justinzobel** [comment]
Yeah if it tied into gnome clocks it'd be great as you'd only setup one
data source.
